modules/virt-enabling-disabling-sidecar-injection.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-managing-sidecar-injection.adoc
+
+[id="virt-enabling-disabling-sidecar-injection_{context}"]
+= Enabling and disabling sidecar injection
+
+When deploying an application into the {ProductName}, you may opt in or out of using sidecar injection by specifying the `sidecar.istio.io/inject` annotation. If the `sidecar.istio.io/inject` annotation is added to the configuration YAML with a value of `"true"`, sidecar injection is enabled. If the annotation is added to the configuration YAML file with a value of `"false"`, sidecar injection is disabled.
+
+NOTE: Opting in ensures that the sidecar injection does not interfere with other OpenShift features such as builder pods used by numerous frameworks within the OpenShift ecosystem.
+
+.Prerequisites
+
+* Identify the deployments for which you want to enable automatic sidecar injection.
+* Locate the application's YAML configuration file.
+
+.Procedure
+
+. Open the application's configuration YAML file in an editor.
+
+. Add `sidecar.istio.io/inject` to the configuration YAML with a value of either `"true"` or `"false"` to enable or disable sidecar injection as illustrated here:
++
+.Sidecar injection enabled example
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: sleep
+  name: sleep
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: sleep
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "true" <1>
+      labels:
+        app: sleep
+    spec:
+      containers:
+      - name: sleep
+        image: tutum/curl
+        command: ["/bin/sleep","infinity"]
+        imagePullPolicy: IfNotPresent
+----
+<1> The enabled sidecar injection annotation
++
+.Sidecar injection disabled example
+[source,yaml]
+----
+apiVersion: cdi.kubevirt.io/v1beta1
+kind: DataVolume
+metadata:
+  name: dv-am
+  annotations:
+      sidecar.istio.io/inject: "false" <1>
+spec:
+  source:
+      http:
+         url: "http://mirrors.nav.ro/fedora/linux/releases/33/Cloud/x86_64/images/Fedora-Cloud-Base-33-1.2.x86_64.qcow2"
```

Comment:
I would not use a specific URL here, since over time actual URLs may change or no longer be valid